common/src/androidTest/java/com/microsoft/identity/common/Util.java

```diff
@@ -48,4 +56,23 @@ static URL getValidRequestUrl() throws MalformedURLException {
         return new URL(Util.VALID_AUTHORITY);
     }
 
+    /**
+     * Utility for adding signatures to a passed PackageInfo in a back-compatible way
+     *
+     * @param packageInfo to add signatures to
+     * @param signatures the signatures to add
+     * @return PackageInfo with signatures added
+     */
+    public static PackageInfo addSignatures(PackageInfo packageInfo, Signature[] signatures) {
```

Comment:
```suggestion
    @Nullable
    public static PackageInfo addSignatures(PackageInfo packageInfo, Signature[] signatures) {
```

Reply:
`addSignatures` basically adds signatures to the passed PackageInfo so why would there be need of flagging it as `@Nullable`?